exercises/tournament/src/test/java/TournamentTest.java

```diff
@@ -1,158 +1,160 @@
-import org.junit.Rule;
+import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.rules.ExpectedException;
 
-import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
 
 public class TournamentTest {
 
-    @Rule
-    public ExpectedException expectedException = ExpectedException.none();
+    private Tournament tournament;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
I don't think this needs `throws Exception`. You should never just throw `Exception` without specifying which exception you're throwing, but in this case you're not throwing any as far as I'm aware, so you don't need it at all :)

Reply:
@FridaTveit I blame intellij for that one :-)

Reply:
Fair enough :)